scripts/netContain/scripts/contivNet.sh

```diff
@@ -133,11 +136,20 @@ if [ $netmaster == true ]; then
 	done
 elif [ $netplugin == true ]; then
 	echo "Starting netplugin"
+    if [[ "$cluster_store" =~ ^etcd://.+ ]]; then
+        store_arg="--etcd-endpoints $(echo $cluster_store | sed s/etcd/http/)"
```

Comment:
We have [another PR to add TLS support to etcd](https://github.com/contiv/netplugin/pull/1077) and I'm working on upgrading the etcd code to v3 which will eventually support TLS as well...  need to think about how to handle this.  I'd like to see us move away from the `datastore://` format entirely

Reply:
we are, i have to do this step by step, the reason it's still here because netmaster doesn't support the new way yet, once netmaster is changed to use new way, these codes can be dropped